TextAnalytics.ipynb

```diff
@@ -1 +1 @@
VCjpN2wK28KjakGY8qoLJ5mw8unMv8OeG+xIEyw1m9lVJt9FKpXrTpKdSBcEIWDNNDN8JnG5mH5U0K3dgcZGqP0hC0spm9gNJm4zhuNPoA7s98La0+7WEVtpzBNfpqG/+jqRDGHN663LGh83s3GTZ9Wo8+HwKMGrLrsfTzwclvQFXDn92nTecqgWsmZ0EnNSUR1Uw8TxsZt9puhNBsDySFqyxaA2WZ1ZMAkx7Mj1CNFVpys6mqYVzHXZuugMTznfprG9uFzCLzanh0pRl1/+XtCbwfjyDdg3gvXXecKoWsAWxeA1m4KOSTgOuZvnwogyCIAgmh2Nx648FZnarpI2AHzXcp5FgZrun/x4taT7JzmYMh54mH9iC6RGbaQAz2xBA0irdljmSVundKqjIA0kI91XA8ckCbs4YjvsWfFy8F9gx7bp/Gqi86TRVIk5B0A9JZwGbAvfRSiGetV6UQRAEQTBuJM0B7m5amGiKfGDvobXDuApeBvdDM3thox2bMAof5UHPBdVJXruvA+4xsx+ljJEXmdkVIz7u0J0KpnIHNghm4MWhehoEQRCME0kfNLMTJJ1Mj902Mzu0gW6NDDP7s6S7JK1nZvc32I+JVR5up3teImkecFBD3Zk4JD0deBawqqS/opVKvAawWmMdm4Ukr91vtT1+EHhwDIeeI2ntLqeCWmvQWMAGs4mFkjYL8ZggCIJgjHw//byN5Sdd9BnAfZJuAX5bPGlmuzbXpenAzG6XtHXT/ZggXgu8DRf1+Wzb84uBDzXRoWDoDN2pIFKIg1mDpO/jVjqh9BcEQRCMlbQo+RCwAa0Ngll5D0oL13axHQHHm9mo1UynDkntNkpzcOuSp5rZaxvq0kQi6c1mdn7T/QhGg6TNaDkVXF13syl2YIPZxOua7kAQBEGw3HIWvqi7h9lv5bZidwqvpFWb6syEs3rb/58ALgFiobYsV0v6LPDy9Pha4Fgz+3WDfQqGxLCdCmIHNgiCIAiCoCaSFpjZ9k33Y5RIejdwCG5t8uO2X60O3GBm+zXSsSlA0ur4jvxjTfdlEpF0PnAvcEZ66q24tsmbmutVMKnEAjYIgiAIgqAmknYC9mYWW7klL8e1gU8AR7b9arGZ/aqZXk02kjYHzgSekp56BDggWYoECUl3mtkWg54LAogU4iAIgiAIgmFwIG7lthJtVm60qX5OOymd89f4Qj3I40vA+8xsPoCkHdJzL22yUxPI7yVtb2YLACRtB/y+4T4FE0osYIMgCIIgCOoTVm5BL55cLF4BzOwaSU9uskMTysHAV9MuP8CjwAEN9ieYYGIBGwRBEARBUJ+wcgt68RNJH8HTiAH2w90SgoSkOcAmZvZiSWsAmNlvGu5WMMFEDWwQBEEQBEFNwsot6IWktYFjgO3wc+I64GgzW9RoxyYMSdeZ2csHvzIIYgEbBEEQBEFQG0nr93rezH4+7r4Ek4OkrYCjWA78geuQdql/D5wD/LZ4PsTBgl7EAjYIgiAIgiAIRoCkHwIfwC1ilvoDR2CjE0k/xUXPOjCzjRroTjDhRA1sEARBEARBEIyGh83sO013YgrYDPcY3h5fyF4PnNpoj4KJJXZggyAIgiAIgmAELA/+wMNA0rnAb4Cz01N7A2uZ2Z7N9SqYVGIHNgiCIAiCIAhGw6z3Bx4Sm5jZi9sez5d0V2O9CSaaWMAGQRAEQRAEwWgIf+A87pC0rZktBJC0DXBDw30KJpRYwAZBEARBEATBaAh/4D5IugffkV4J2F/S/enx+kB8Z0FPogY2CIIgCIIgCEZA+AP3Zyb7qYJQaw56EQvYIAiCIAiCIBgB4Q8cBMMnFrBBEARBEARBEATBVDCn6Q4EQRAEQRAEQRAEQQ6xgA2CIAiCIAiCIAimgljABkEQBMGIkHSUpPsk3S3pTknbSHqvpNWa7lsQBEEQTCNRAxsEQRAEI0DSS4DPAjuY2RJJ6wBPAm4EtjKzRxrtYBAEQRBMIbEDGwRBEASj4RnAI2a2BCAtWPcAngnMlzQfQNJrJN0k6XZJ50mam57/maSPp9/dJmmepMsl/VjSwek1z5B0XdrdvVfSy5r5qEEQBEEwHmIHNgiCIAhGQFqILgBWA64CzjGzayX9jLQDm3ZlvwXsbGa/lXQEsLKZHZted7yZnSLpc8BOwHbAKsB9ZvY0Se8HVjGz4yStAKxmZovH/mGDIAiCYEys2HQHgiAIgmA2YmaPSdoSeBmwI3COpCO7XrYtsBlwgyTwFOOb2n5/Ufp5DzA3LU4XS/qDpLWAW4F/kbQS8G0zu3N0nygIgiAImicWsEEQBEEwIszsT8A1wDWS7gEO6HqJgCvNbO8Z3mJJ+vnntv8Xj1c0s+skvRx4A3CmpE+Z2VeH9gGCIAiCYMKIGtggCIIgGAGSNpH0vLantgB+DiwGVk/PLQS2k/Tc1GY1Sc8vcYz1gYfM7MvAV4B5Q+l8EARBEEwosQMbBEEQBKNhLnBySvV9AvhP4G+BvYFLJT1oZjtKehvwdUkrp3YfBv4j8xg7AIdLehx4DNh/iP0PgiAIgokjRJyCIAiCIAiCIAiCqSBSiIMgCIIgCIIgCIKpIBawQRAEQRAEQRAEwVQQC9ggCIIgCIIgCIJgKogFbBAEQRAEQRAEQTAVxAI2CIIgCIIgCIIgmApiARsEQRAEQRAEQRBMBbGADYIgCIIgCIIgCKaCgTCm0AAAAAZJREFU/wNAv1CB5oDHLgAAAABJRU5ErkJggg==\n","text/plain":["<Figure size 1152x648 with 1 Axes>"]},"metadata":{"tags":[],"needs_background":"light"}}]},{"cell_type":"code","metadata":{"trusted":false,"id":"3Z0P4Lg6ih31","colab_type":"code","colab":{}},"source":[""],"execution_count":0,"outputs":[]}]}
```

Comment:
@eizaburo-doi can you elaborate on what changes did you add? GitHub doesn't display Python Notebook that good. (h/t lecture video)

Reply:
Added more example sentences to be evaluated.